ansible/roles/python-devel/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+# tasks file for roles/python-devel
+
+- name: check default python version
+  shell: "python --version 2>&1 | awk '{print $2}'"
+  register: python_installed_version
```

Comment:
We should be in full control of these machines, so the python verison should be known.